### DISCUSSION TITLE
Is requestSubmit supported?

### DISCUSSION THREAD

**Discussion by khmyznikov** [calebdwilliams/element-internals-polyfill#80] [Q&A]
I'm making a custom submit button, do your polyfill supports this https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/requestSubmit ?

**Comment by calebdwilliams** [comment]
There are separate polyfills for that particular feature. There’s a similar function I wrote for `@open-wc/form-helpers` https://github.com/open-wc/form-participation/tree/main/packages/form-helpers#readme

**Reply by khmyznikov** [reply]
Do you recommend something like this? https://github.com/javan/form-request-submit-polyfill not sure I understood your answer.
